ouroboros-consensus-test/src/Test/Util/MockDiffSeq.hs

```diff
@@ -0,0 +1,196 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE DerivingVia                #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase                 #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE RankNTypes                 #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE StandaloneKindSignatures   #-}
+{-# LANGUAGE TupleSections              #-}
+
+module Test.Util.MockDiffSeq (
```

Comment:
Maybe we should explain what is the role of the mock diff in the benchmarks.

Reply:
My guess: it serves as the naïve implementation of diff sequences. But if I'm right, why do we need fingertrees at all :thinking:

Reply:
This is removed in latter commits.